CMP1903_A1_2324/Game.cs

```diff
@@ -17,5 +17,30 @@ internal class Game
 
         //Methods
 
+        //these are three object instantiaons of die objects. They are private as they do not need to be accessed outside of the game class
+        private Die _die1 = new Die(); 
```

Comment:
Instead of 3 dice could potentially roll one die 3 times by iterating through the rollDice method of the Die class.